Mage/Logger/Writers/ConsoleWriter.cs

```diff
@@ -56,7 +56,7 @@ public class ConsoleWriter {
 		Logger.logEmitter.on ("critical", (object sender, LogEntry logEntry) => {
 			Debug.LogError (makeLogString("critical", logEntry.context, logEntry.message));
 			if (logEntry.data != null) {
-				if (logEntry.data is Exception) {
+				if (logEntry.data is Exception && (logEntry.data as Exception).StackTrace != null) {
```

Comment:
What do you do if it's an exception still? The logEntry.data passed to LogError on line 63 will do the trick?